TownHallTriviaApp/main/templates/main/registerTeam.html

```diff
@@ -4,19 +4,20 @@
   <head>
     <title>Town Hall Trivia</title>
     <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
-  <body>
+  <body class="body">
     <header>
       <div class="container">
       <h1 class="logo">Town Hall Trivia</h1>
     </header>
     <h1>Team Registration</h1>
-    <h2>{{ message }}</h2>
-    <h3>Game Id: {{ gameId }}</h3>
```

Comment:
I was thinking keeping the gameId visible at all times. I feel like that''l help people not get confused.

Reply:
Respectfully declining this suggestion